docs/usage/options.md

```diff
@@ -50,6 +50,8 @@ Available Options for the IONOS Cloud Docker Machine Driver:
 | `--ionoscloud-create-nat`               | Create a new NAT with some default open ports                                                                                                                                                 |
 | `--ionoscloud-nat-public-ips`           | If --ionoscloud-create-nat is set, change the NAT's public IPs to these values                                                                                                                |
 | `--ionoscloud-nat-lans-to-gateways`     | If --ionoscloud-create-nat is set, change the NAT's mappings of LANs to Gateway IPs to these values. Must respect format `1=10.0.0.1,10.0.0.2:2=10.0.0.10`                                    |
+| `--ionoscloud-nat-flowlogs`     | If --ionoscloud-create-nat is set, add flowlogs to the nat. Must respect format `name:action:direction:bucket`,                                    |
+| `--ionoscloud-nat-rules`     | If --ionoscloud-create-nat is set, add rules to the NAT. Must respect format `name:type:protocol:public_ip:source_subnet:target_subnet:target_port_range_start:target_port_range_end`, to skip providing an optional value just omit it (`name:type:protocol::source_subnet:::`), not setting public IP will use the public IP of the NAT for the rule                                    |
```

Comment:
if I want to add multiple rules, then I would separate these models via `,` ?

Reply:
you should add another --ionoscloud-nat-rules with a second rule and an array will be formed
using "," would result in an error if i remember correctly as the whole array is passed as a string